composer.json

```diff
@@ -10,10 +10,10 @@
         }
     ],
     "require": {
-        "php": "^8.0",
-        "illuminate/support": "^9.0",
```

Comment:
I would allow to still use it with 9.0, i.e change it to ^9.0|^11.0

Reply:
fixed